docs/source/configuring_deploying.rst

```diff
@@ -307,3 +307,117 @@ setting has been updated to no longer include the old node's public key, the
 leader will stop sending messages to the node and ignore any messages from it.
 Once this happens, the node can be shutdown safely since it is no longer
 participating in the network.
+
+
+Reading the Logs/Output
+=======================
```

Comment:
The "Reading the Logs/Output" content should be merged into the content above where appropriate, instead of starting a new section here. For example, you could merge the content about the Raft process starting but not actually connecting into the "Connecting to the validator" section of "Starting a Raft Engine".